examples/textures/textures_particles_blending.php

```diff
@@ -0,0 +1,169 @@
+        1.0,
+        (float) $raylib->getRandomValue(1, 30) / 20.0,
+        (float) $raylib->getRandomValue(0, 360),
+        false
+    ) {
+        public Vector2 $position;
+        public Color $color;
+        public float $alpha;
+        public float $size;
+        public float $rotation;
+        public bool $active;    // NOTE: Use it to activate/deactive particle
+
+        public function __construct(
+            Vector2 $position,
+            Color $color,
+            float $alpha,
+            float $size,
+            float $rotation,
+            bool $active
+        ) {
+            $this->position = $position;
+            $this->color = $color;
+            $this->alpha = $alpha;
+            $this->size = $size;
+            $this->rotation = $rotation;
+            $this->active = $active;
+        }
+    };
+}
+
+$gravity = 3.0;
+
+$smoke = $raylib->loadTexture(__DIR__ . '/resources/spark_flame.png');
+
+$blending = HasRaylibBlendModeConstants::BLEND_ALPHA;
+
+$raylib->setTargetFPS(60);
+//-------------------------------------------------------------------------------------
+
+// Main game loop
+while (!$raylib->windowShouldClose()) { // Detect window close button or ESC key
+    // Update
+    //----------------------------------------------------------------------------------
+
+    // Activate one particle every frame and Update active particles
+    // NOTE: Particles initial position should be mouse position when activated
+    // NOTE: Particles fall down with gravity and rotation... and disappear after 2 seconds (alpha = 0)
+    // NOTE: When a particle disappears, active = false and it can be reused.
+    $i = 0;
+    for (; $i < MAX_PARTICLES; $i++) {
+        if (!$mouseTail[$i]->active) {
+            $mouseTail[$i]->active = true;
+            $mouseTail[$i]->alpha = 1.0;
+            $mouseTail[$i]->position = $raylib->getMousePosition();
+            $i = MAX_PARTICLES;
+        }
+    }
+
+    for ($i = 0; $i < MAX_PARTICLES; $i++) {
+        if ($mouseTail[$i]->active) {
+            $mouseTail[$i]->position->y += $gravity / 2;
+            $mouseTail[$i]->alpha -= 0.005;
+
+            if ($mouseTail[$i]->alpha <= 0.0) {
+                $mouseTail[$i]->active = false;
+            }
+
+            $mouseTail[$i]->rotation += 2.0;
+        }
+    }
+
+    if ($raylib->isKeyPressed(HasRaylibKeysConstants::KEY_SPACE)) {
+        if ($blending == HasRaylibBlendModeConstants::BLEND_ALPHA) {
+            $blending = HasRaylibBlendModeConstants::BLEND_ADDITIVE;
+        } else {
+            $blending = HasRaylibBlendModeConstants::BLEND_ALPHA;
+        }
+    }
+    //----------------------------------------------------------------------------------
+
+    // Draw
+    //----------------------------------------------------------------------------------
+    $raylib->beginDrawing();
+    $raylib->clearBackground(Color::darkGray());
+
+    $raylib->beginBlendMode($blending);
+
+    // Draw active particles
```

Comment:
[The original code indents everything between `BeginBlendMode()` and `EndBlendMode()`](https://github.com/raysan5/raylib/blob/3.5.0/examples/textures/textures_particles_blending.c#L107-L114). Let's please follow the same pattern here.

You can add `phpcs:disable` comments to disable indentation checks for this specific block. See this file for example: https://github.com/nawarian/raylib-ffi/blob/main/examples/textures/textures_blend_modes.php#L54-L122